decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super
+    @namable = nameable
+  end
+
+  def correct_name
+    @nameable
+  end
+end
```

Comment:
- [ ] You seem to be missing an attribute accessor for nameable. Kindly include it. ``` attr_accessor :nameable```

Reply:
Hello @Nemwel-Boniface  👋 

Thanks for the review. with respect to the error you are getting, when you run those tests on the code: it is coming from the number of parameters provided at the instance of the class.

Provide 3 parameters and the error would go.